lib/muser_ctx.c

```diff
@@ -1011,6 +1011,10 @@ static int prepare_ctx(lm_ctx_t *lm_ctx)
     uint32_t max_ivs = 0, i;
     size_t size;
 
+    if (lm_ctx->ready != 0) {
```

Comment:
In which case is `prepare_ctx` redone?

Reply:
Caller can call lm_ctx_drive() multiple times.

Reply:
How about introducing a new function, e.g. `lm_ctx_realize`, that does all this work and dropping `lm_ctx->ready`?
So it would look like this:
```
lm_ctx_create(...)
lm_ctx_setup_region(...)
lm_ctx_realize(...)
while (true)
    lm_ctx_drive(...)
```

Reply:
Good idea, although if that's where the accept() will now happen, do we want a clearer name to indicate it will block/wait for the client? Or just docs?

Reply:
I also liked the idea. But still we may need to mark context realised/ready, as there are chances caller can call  lm_ctx_realize() multiple times. Or we do not care about it.